Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -785,4 +794,24 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         alert.addAction(.init(title: L10n.Alert.Actions.ok, style: .default, handler: { _ in }))
         present(alert, animated: true)
     }
+    
+    // MARK: - InputTextViewImagePasteDelegate
+    
+    open func inputTextView(_ inputTextView: InputTextView, didPasteImage image: UIImage) {
+        do {
+            let imageUrl = try image.temporaryLocalFileUrl()
+            let urlAndType: (URL, AttachmentType) = (imageUrl!, .image)
```

Comment:
- Are you sure the `!` is safe here?
- Why do you create this tuple? It's not used in the code as a tuple anywhere.

Reply:
Fixed the force unwrap, it wasn't safe in that context. Also removed the tuple